package.json

```diff
@@ -29,7 +29,6 @@
     "superagent-retry": "git://github.com/ajacksified/superagent-retry#d39d7adbcd021d8ed09df440dba970c91fed9cd4"
   },
   "devDependencies": {
-    "babel": "^5.0.0",
```

Comment:
:no_bell: updating to babel 6 as we did on reddit-mobile later for those playing along at home